### DISCUSSION TITLE
35 second delay before playback

### DISCUSSION THREAD

**Discussion by pcwii** [snapcast/snapcast#880] [Q&A]
I am trying to troubleshoot a playback delay on my new setup and not sure where to begin.
My setup is as follows.
I have a snapcast server running in docker on computer #1 (homeserver.local).
I have setup tcp streaming from computer  #2 using `parec --rate=48000 --format=s16le --channels=2 | nc homeserver.local 4956`
I have snapcast android client running on phone #1

Now if I start audio playback on computer #2 it will take ~35 seconds before the audio begins to play on phone #1.

Is there a buffer setting on the server side that could be causing this?
Thanks for a great piece of software!

**Comment by artpelling** [comment]
Default `netcat` buffer size is 4MB I think which amounts to more than 20s. Maybe try something smaller like 4KB?

**Reply by artpelling** [reply]
I've created a gist with things to try https://gist.github.com/artpelling/c97530b979c9e7f781c28bc1c433780d
